src/defer-load.directive.ts

```diff
@@ -20,19 +21,21 @@ export class DeferLoadDirective implements AfterViewInit, OnDestroy {
         @Inject(PLATFORM_ID) private platformId: Object
     ) { }
 
-    public ngAfterViewInit () {
-        if (isPlatformBrowser(this.platformId)) {
+    public ngOnInit () {
```

Comment:
slightly concerned about this change.
are you sure this does not cause any issues in the browser ?
there is a reason this was initially in ngAfterViewInit. Would be great to verify that everything still works when we move it to ngOnInit.